rules/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromStrictScalarReturnExprRector.php

```diff
@@ -117,6 +118,11 @@ public function refactorWithScope(Node $node, Scope $scope): ?Node
             return null;
         }
 
+        // skip null as often placeholder value and not an only type
+        if ($scalarReturnType instanceof NullType) {
```

Comment:
this should be fine on Function_, Closure, and when class is final, so that seems need to be checked

I will add new PR for that.

Reply:
Oh, I see what you mean about placeholder, seems ok then